pyopensprinkler/const.py

```diff
@@ -56,3 +56,13 @@
 START_TIME_SUNSET_BIT = 13
 START_TIME_SUNRISE_BIT = 14
 START_TIME_MINUTES_MASK = 0x7FF
+
+WEEKDAYS = [
```

Comment:
A bit pedantic but "weekdays" to me is Monday to Friday, maybe we should use `DAYS_OF_WEEK` ot `DAYS_OF_THE_WEEK`

Reply:
I guess this is due to the OS API itself, maybe its fine to just leave it.

Reply:
Agreed on both points. I try to follow existing conventions as much as possible.